discovery_engine_core/web-api/openapi.yaml

```diff
@@ -0,0 +1,153 @@
+openapi: 3.0.3
+info:
+  title: Personalization API
+  version: 1.0.0-rc1
+  description: |-
+    The endpoints can be contacted at `t<name>.xaynet.dev/v1` or `t<name>.xaynet.dev/v1.0`.
+    The `v1` path will always point to the latest `v1.x`.
+    
+    # Auth
+    To authenticate with the API two bearer tokens will be provided. One will work for the
+    personalization endpoints and the other one will allow to add documents to the system.
+    
+tags:
+  - name: personalization
+    description: Interaction of the user and personalized document
+  - name: ingestion
+    description: Add documents to the system
+paths:
+  /users/{user_id}/documents/personalized:
```

Comment:
I don't really understand this namesapce tbh: 

`userid/documents` 

From the customers perspective documents and personalized is is basically the name so we can shorten this
to

`userID/personalizedDocuments` or just `documents`

Reply:
Documents for the user are what they load on the system and are the basic items that we can match to the users.
`users/{id}/documents` could be later use to return which documents the user interacted with or something similar.